### PR TITLE
Add configuration parameters for RSUHealthMonitorPlugin

### PR DESCRIPTION
**Pull request by dan-du-car**
# PR Details
## Description
Add configuration parameter to set SNMP security level.

## Related Issue
https://github.com/usdot-fhwa-stol/cda-telematics/issues/213

## JIRA
https://usdot-carma.atlassian.net/browse/VH-1316
## Motivation and Context
Telematic tool integration test

## How Has This Been Tested?
Unit test

## Types of changes

- [x] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[V2XHUB Contributing Guide](https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/v2i-hub/RSUHealthMonitorPlugin/manifest.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
         },
         {
             "key":"RSUConfigurationList",
-            "default":"{ \"RSUS\": [ { \"RSUIp\": \"192.168.XX.XX\", \"SNMPPort\": \"161\", \"AuthPassPhrase\": \"dummy\", \"User\": \"authOnlyUser\", \"RSUMIBVersion\": \"RSU4.1\" },{ \"RSUIp\": \"192.168.00.XX\", \"SNMPPort\": \"162\", \"AuthPassPhrase\": \"tester\", \"User\": \"authPrivUser\", \"RSUMIBVersion\": \"RSU4.1\" }] }",
+            "default":"{ \"RSUS\": [ { \"RSUIp\": \"192.168.XX.XX\", \"SecurityLevel\":\"authPriv\", \"SNMPPort\": \"161\", \"AuthPassPhrase\": \"dummy\", \"User\": \"authOnlyUser\", \"RSUMIBVersion\": \"RSU4.1\" ,\"RSUIdentifier\":\"NA\"},{ \"RSUIp\": \"192.168.00.XX\", \"SecurityLevel\":\"authPriv\", \"SNMPPort\": \"162\", \"AuthPassPhrase\": \"tester\", \"User\": \"authPrivUser\", \"RSUMIBVersion\": \"RSU4.1\" ,\"RSUIdentifier\":\"NA\"}] }",
```
Comment (paulbourelly999) [inline src/v2i-hub/RSUHealthMonitorPlugin/manifest.json:22]: Is RSU Name or ID not the identifier? I think the RSU spec has snmp endpoints for both.

File: src/v2i-hub/RSUHealthMonitorPlugin/src/RSUHealthMonitorPlugin.cpp [hunk 2]
```diff
@@ -27,6 +27,7 @@ namespace RSUHealthMonitor
         for (auto rsuConfig : _rsuConfigListPtr->getConfigs())
         {
             auto rsuStatusJson = _rsuWorker->getRSUStatus(rsuConfig.mibVersion, rsuConfig.rsuIp, rsuConfig.snmpPort, rsuConfig.user, rsuConfig.authPassPhrase, rsuConfig.securityLevel, SEC_TO_MICRO);
+            rsuStatusJson[RSUIdentifierKey] = rsuConfig.RSUIdentifier;
```
Comment (paulbourelly999) [inline src/v2i-hub/RSUHealthMonitorPlugin/src/RSUHealthMonitorPlugin.cpp:30]: Why are we pulling this from the configuration, are we not setting this on the RSU?

**Review by paulbourelly999 — CHANGES_REQUESTED** [review]
One issue I have with the current setup is that is seems we are creating our own RSU Identifier instead of using the existing RSU name and/or ID fields both of which can be set and retrieved from the RSU via SNMP. 

@paulbourelly999  Removed

File: src/v2i-hub/RSUHealthMonitorPlugin/manifest.json (hunk 1 above)
Comment (dan-du-car) [reply src/v2i-hub/RSUHealthMonitorPlugin/manifest.json:22]: RSUs specs does have RSU Name and ID. However, they usually have the same values across RSUs. We could use the RSUs specs, and that requires to update each individual RSUs setup either via SNMP request or some other ways. For the telematic tool purpose, I assume this is the least effort to meet the telematic requirement.

File: src/v2i-hub/RSUHealthMonitorPlugin/src/RSUHealthMonitorPlugin.cpp (hunk 2 above)
Comment (dan-du-car) [reply src/v2i-hub/RSUHealthMonitorPlugin/src/RSUHealthMonitorPlugin.cpp:30]: This is to directly sent as part of the RSU status payload.

**Review by paulbourelly999 — APPROVED** [review]

**Pull request merged**